lib/build.gradle

```diff
@@ -87,12 +91,17 @@ publishing {
         release(MavenPublication) {
             groupId = "space.taran"
             artifactId = "arklib"
-            version = "0.2.2"
```

Comment:
I think we should keep it. For branch builds it's better to use local machine, it's also faster than wait 40-50 minutes on CI.

Reply:
Added version